src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 1.5;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      font-weight: $font-weight-regular;
+      line-height: 14px;
+      text-align: left;
+      color: $text-color;
+    }
+  }
+
+  &__rating {
+    box-sizing: border-box;
+    width: $card-content-width-adjustment;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: $padding-inline;
+    margin-bottom: 24px;
```

Comment:
The margin-bottom for '.product-card__rating' is hard-coded in pixels. It may be more maintainable to use variables or relative units for spacing to ensure consistent spacing across different components.